pkg/plugin/run.go

```diff
@@ -359,11 +357,13 @@ func (e *AutoscaleEnforcer) updateMetricsAndCheckMustMigrate(
 	node *nodeState,
 	metrics *api.Metrics,
 ) bool {
-	// This pod should migrate if (a) we're looking for migrations and (b) it's next up in the
-	// priority queue. We will give it a chance later to veto if the metrics have changed too much
+	// This pod should migrate if (a) it's allowed to migrate, (b) we're looking for migrations and
```

Comment:
What does (b) mean?

Reply:
I clarified the language. It now reads:

> This pod should migrate if (a) it's allowed to migrate, (b) node resource usage is high enough that we should migrate \*something\*, and (c) it's next up in the priority queue. We will give it a chance later to veto if the metrics have changed too much.

LMK if that makes more sense.